tests/test.py

```diff
@@ -173,80 +284,68 @@ def test_currentmount(self):
         slow_exe(["rm", str(link_file)])
 
         f.finish()
-        f_json.finish()
 
         cwd = str(self.tmp_path)
         cwd_re = re.escape(cwd)
         test_file_str = str(test_file)
 
         # file creation
-        f.assert_log(rf"^touch.*\sC?W?O\s+{re.escape(test_file_str)}")
-        f.assert_log(rf"^touch.*\sC?WO?\s+{re.escape(test_file_str)}")
-        f.assert_log(rf"^bash.*\sC?WO?\s+{re.escape(test_file_str)}")
-
-        f_json.assert_json(lambda e: e["comm"] == "touch" and e["path"] == test_file_str and "O" in e["types"])
-        f_json.assert_json(lambda e: e["comm"] == "touch" and e["path"] == test_file_str and "W" in e["types"])
-        f_json.assert_json(lambda e: e["comm"] == "bash" and e["path"] == test_file_str and "W" in e["types"])
+        f.assert_log(lambda e: e["comm"] == "touch" and e["path"] == test_file_str and "O" in e["types"],
+                     rf"^touch.*\sC?W?O\s+{re.escape(test_file_str)}")
```

Comment:
FTR, I like these assertions. They are explicit in how both outputs are expected to look like. :+1: